application/common/components/Emailer.php

```diff
@@ -323,6 +329,7 @@ public function init()
             EmailLog::MESSAGE_TYPE_PASSWORD_EXPIRING => $this->subjectForPasswordExpiring,
             EmailLog::MESSAGE_TYPE_PASSWORD_EXPIRED => $this->subjectForPasswordExpired,
             EmailLog::MESSAGE_TYPE_PASSWORD_PWNED => $this->subjectForPasswordPwned,
+            EmailLog::MESSAGE_TYPE_UNUSED_USERS => $this->subjectForUnusedUserAccounts,
```

Comment:
I suggest defining this constant (`MESSAGE_TYPE_UNUSED_USERS`) here in `Emailer`. See comment in `EmailLog.php` for explanation.